types/sinon/index.d.ts

```diff
@@ -110,6 +114,9 @@ declare namespace Sinon {
         resolves(value?: any): SinonStub;
         throws(type?: string): SinonStub;
         throws(obj: any): SinonStub;
+        throwsArg(index: number): SinonStub;
+        throwsException(type?: string): SinonStub;
```

Comment:
Not sure if we should add this here as in fact it's not working (see https://github.com/sinonjs/sinon/issues/922) and considered to be changed.